src/Pages/suyeonkim/Main/Aside/RecoPerson/RecoPerson.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import RecoLi from './RecoLi/RecoLi';
+import './RecoPerson.scss';
+
+class RecoPerson extends React.Component {
```

Comment:
```suggestion
class RecommendedPerson extends React.Component {
```
애매하게 줄이기보다 정확히 써주는 편이 더 좋아보입니다!